test/model_level_tests/models/TF_OV/run_infer_single.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+# This script is used to fetch/run models from a curated model-repo for OV-IE integration project
+# Invoke locally: .../run_infer_single.sh resnet_50 bike 'mountain bike, all-terrain bike'
+
+MODEL=$1
+if [ "${BUILDKITE}" == "true" ]; then
+    echo "--- Model: ${MODEL}"
+fi
+if [ "${MODEL}" == "" ]; then
+    echo "Error: model not specified!" && exit 1
+fi
+IMAGE=$2
+if [ "${IMAGE}" == "" ]; then
+    echo "Error: image not specified!" && exit 1
+elif [[ ! ${IMAGE} =~ *.* ]]; then
+    IMAGE="${IMAGE}.jpg"
+fi
+INFER_PATTERN=$3
+if [ "${INFER_PATTERN}" == "" ]; then
+    echo "Error: expected pattern not specified!" && exit 1
+fi
```

Comment:
Can we use a default for this if one isn't specified?

Reply:
I thought about it, but I think it's best for the developer to use args/params as needed. There is an example at top of the file.